packages/apps/transfer/src/services/key-pairs/index.ts

```diff
@@ -0,0 +1,15 @@
+import { genKeyPair } from '@kadena/cryptography-utils';
+import { IKeyPair } from '@kadena/types';
+
+import { downloadFileToBrowser } from '../utils/file-download';
+
+export function generateKeyPair(): IKeyPair {
+  return genKeyPair();
+}
+
+export function downloadKeyPairToBrowser(keyPair: IKeyPair): void {
```

Comment:
since there is nothing else to generate we can just put keypair generator inside this function, cause it's meant to only download the input